maintainers/maintainer-list.nix

```diff
@@ -4666,6 +4666,12 @@
     githubId = 494012;
     name = "Kevin Cox";
   };
+  kevingriffin = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add kevingriffin``.

Reply:
I've made this change.